src/jio-footer.ts

```diff
@@ -200,6 +200,18 @@ export class Footer extends LitElement {
          </div>
       </div>
    </div>
+   <div class="container">
+     <div class="footer-content">
```

Comment:
```suggestion
     <div class="row">
```

Reply:
This change would align the bottom text with the ones up top in the footer I have checked